cookbook/ui_customization/create_a_new_field_type.rst

```diff
@@ -0,0 +1,146 @@
+How to add a new field type
+===========================
+
+If you want a custom rendering for one of your attribute, attribute type or reference data you need to create a new field type. In this cookbook, we will got through each steps needed to create a custom field type.
```

Comment:
one of your attributeS, attribute type or reference data, you...
we will go through each step